programmers/실패율.js

```diff
@@ -0,0 +1,27 @@
+function solution(N, stages) {
+    const result = [];
+    let players = stages.length;
+    
+    for (let i = 0; i < N; i++) {
+        let notClear = stages.filter(p => p === i + 1).length;
+        let failureRate = 0;
+        
+        if (notClear === 0) {
+            failureRate = 0
+        } else {
+            failureRate = notClear / players;
+        }
+        players = players - notClear;
+        
+        result.push({ N: i + 1, failureRate: failureRate});
+    }
+    
+    result.sort((a, b) => {
+        if (a.failureRate !== b.failureRate) {
+            return b.failureRate - a.failureRate;
+        } else {
+            return a.failureRate - b.failureRate;
+        }
+    })
```

Comment:
```suggestion
    result.sort((a, b) => {
        if (a.failureRate !== b.failureRate) return b.failureRate - a.failureRate;
        return a.failureRate - b.failureRate;
    })
```
여기도 이렇게 하면 줄 수를 약간 줄일 수 있을 것 같습니다!
저보다 훨씬 잘 푸셨네요ㅠㅠ